### DISCUSSION TITLE
Cut/copy and paste don’t work in iOS

### DISCUSSION THREAD

**Discussion by chegeiser** [standardnotes/forum#2580] [General]
Until one of the past 1 or 2 updates cut/copy and paste worked, as it works in iOS in general. It no longer works. A long press, or double press, on a word does not bring up a context menu for choosing cut/copy and paste.

1. Double tap, or king press on the screen.
2. No context menu pops up giving cut/paste options. 

Cut/copy and paste should work as the do in iOS normally. 

 - iPhone 13 mini
 - OS: iOS 15.4 and 15.5
 - Standard Notes app
 - Version 3.14.0 (117)

**Comment by atmoio** [comment]
Is this using a particular editor, or the plaintext editor?

**Comment by chegeiser** [comment]
I hadn’t thought to try other editors. This is with Markdown Pro. I just tried the plain editor and it works correctly. 

Works in:
 * Plain
 * Markdown Basic
 * Markdown Math
 * Plus 
 * Bold
 * Task
 
Doesn’t work in:
 * Markdown Minimist
 * Markdown Pro
 * Code

**Comment by atmoio** [comment]
Hmm the last 3 editors are all CodeMirror based. Wonder what's going on..

**Comment by BobWs** [comment]
I’m having the same problem…
